pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -246,18 +257,22 @@ func (e *BootstrapSigner) getConfigMap() *v1.ConfigMap {
 		return nil
 	}
 
-	if exists {
-		return configMap.(*v1.ConfigMap)
-	}
-	return nil
+	return configMap
+
 }
 
 func (e *BootstrapSigner) listSecrets() []*v1.Secret {
-	secrets := e.secrets.List()
+	secrets, err := e.secretLister.Secrets(e.secretNamespace).List(labels.Everything())
+	if err != nil {
+		utilruntime.HandleError(err)
+		return nil
```

Comment:
Is this ok? It means that `getTokens` gets no secrets, which means that `signConfigMap` gets no tokens. Is that alright, or does the error need to be propagated up?

Reply:
Yes, we need just as `getConfigMap` when there has an error, It is also keep as the original logic

Reply:
There is a difference, though, between the old code and this PR. Before, it was impossible for `e.secrets.List()` to return an error. Now, when using `secretLister`, you may get an error. Just wanted to point that out.